packages/core/contracts/financial-templates/common/SyntheticTokenExclusiveMinter.sol

```diff
@@ -0,0 +1,77 @@
+pragma solidity ^0.6.0;
+import "../../common/implementation/ExpandedERC20ExclusiveMinter.sol";
+import "../../common/implementation/Lockable.sol";
+
+
+/**
+ * @title Burnable and mintable ERC20 with only one minter allowed.
+ * @dev The contract deployer will initially be the only minter, burner and owner capable of adding new roles.
+ */
+
+contract SyntheticTokenExclusiveMinter is ExpandedERC20ExclusiveMinter, Lockable {
```

Comment:
Similar to `ExpandedERC20ExclusiveMinter`, this contract copies `SyntheticToken` but changes `addMinter` to `resetMinter`.

Reply:
If this is the implementation, why do we need `ExpandedERC20ExclusiveMinter` at all? Could we combine both implementations and put them into this contract or is `ExpandedERC20ExclusiveMinter` used elsewhere?